### PR TITLE
feature: update app layout

### PR DESCRIPTION
**Pull request by sveta-leonovets-twi**

### CODE REVIEW

File: src/components/AppContainer.tsx [hunk 1]
```diff
@@ -88,6 +91,7 @@ const AppContainer: React.FC = () => {
     removeConversation,
     updateCurrentConversation,
       addNotifications,
+    logout
```
Comment (berkus) [inline src/components/AppContainer.tsx:94]: indents?

Comment (berkus) [reply src/components/AppContainer.tsx:94]: image

File: src/components/AppContainer.tsx [hunk 2]
```diff
@@ -115,10 +119,10 @@ const AppContainer: React.FC = () => {
         });
 
         handlePromiseRejection(async () => {
-            if (conversation.status === "joined") {
-              const result = await getConversationParticipants(conversation);
+          if (conversation.status === "joined") {
+            const result = await getConversationParticipants(conversation);
               updateParticipants(result, conversation.sid);
-            }
+          }
```
Comment (berkus) [inline src/components/AppContainer.tsx:125]: indents?

image

File: src/components/AppContainer.tsx [hunk 3]
```diff
@@ -176,8 +180,6 @@ const AppContainer: React.FC = () => {
       });
 
         client.addListener("tokenExpired", () => {
-          const username = localStorage.getItem("username");
-          const password = localStorage.getItem("password");
           if (username && password) {
             getToken(username, password).then((token) => {
               login(token);
```
Comment (berkus) [inline src/components/AppContainer.tsx:185]: If no username and password in local storage - should navigate to sign in screen? Client will be stuck unusable here.

File: src/components/AppContainer.tsx (hunk 1 above)
Comment (sveta-leonovets-twi) [reply src/components/AppContainer.tsx:94]: prettier fixed that way

File: src/components/AppContainer.tsx (hunk 3 above)
Comment (sveta-leonovets-twi) [reply src/components/AppContainer.tsx:185]: hm no, they were just moved up to component level

Comment (sveta-leonovets-twi) [reply src/components/AppContainer.tsx:185]: https://github.com/twilio/twilio-conversations-demo-react/pull/3/files/70d13de1fb9e9c0f8ca423675f975acc13d1ab26#diff-820ddeb4092719dc94913a9737da496221ee7a2e961440e7dbe18c8c0a8dfdd8R77

Comment (berkus) [reply src/components/AppContainer.tsx:185]: We went looking everywhere, but couldn’t find those commits.

Comment (sveta-leonovets-twi) [reply src/components/AppContainer.tsx:185]: @berkus new link 
https://github.com/twilio/twilio-conversations-demo-react/pull/3/commits/d9484da4e4892b598abc273db7d8da341160d045#diff-820ddeb4092719dc94913a9737da496221ee7a2e961440e7dbe18c8c0a8dfdd8R77

same file `src/components/AppContainer.tsx` 77 line

File: src/components/AppContainer.tsx (hunk 2 above)
Comment (sveta-leonovets-twi) [reply src/components/AppContainer.tsx:125]: prettier

**Review by InfiniteRain — APPROVED** [review]
some weird indentation by prettier here and there, but i think it's fine.

**Pull request merged**